### PR TITLE
add hystrix example

### PR DESCRIPTION
**Pull request by EvenLjj**
add hystrix example

### CODE REVIEW

File: example/src/test/java/com/alipay/sofa/rpc/hystrix/HelloServiceFallback.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+public class HelloServiceFallback implements HelloService {
+
+    @Override
+    public String sayHello(String name, int age) {
+        return "fallback " + name + " from server! age: " + age;
+    }
+
+    @Override
+    public String sendMsg(String msg, Integer waitTime) {
+        return "fallback msg from server!: " + msg;
```
Comment (OrezzerO) [inline example/src/test/java/com/alipay/sofa/rpc/hystrix/HelloServiceFallback.java:34]: Fallback is not invoked by server. It is invoked by client.

File: example/src/test/java/com/alipay/sofa/rpc/hystrix/HystrixClientMain.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        // 也可以自定义 FallbackFactory 直接注入 FallbackFactory
+        SofaHystrixConfig.registerFallbackFactory(consumerConfig, new HelloServiceFallbackFactory());
+
+        HelloService helloService = consumerConfig.refer();
+
+        String rpcResp = helloService.sendMsg("test", -1);
+        LOGGER.info(rpcResp);
+
+        String hystrixResp = helloService.sendMsg("timeoutTest", 3000);
+        LOGGER.info(hystrixResp);
```
Comment (OrezzerO) [inline example/src/test/java/com/alipay/sofa/rpc/hystrix/HystrixClientMain.java:62]: It's hard to find what is printed by LOGGER. `System.out.println()` may be better.

Comment (nobodyiam) [reply example/src/test/java/com/alipay/sofa/rpc/hystrix/HystrixClientMain.java:62]: Is this a demo or test? If it is a test, I suppose we need to use assert to test the result.

Comment (EvenLjj) [reply example/src/test/java/com/alipay/sofa/rpc/hystrix/HystrixClientMain.java:62]: There is already a test for this. This is a demo. The logger is also used in other demos. Can use the class name to search.

**Pull request closed without merging**